apps/vr-tests-react-components/.storybook/main.js

```diff
@@ -22,7 +22,16 @@ module.exports = {
 
     config.module.rules.unshift({
       test: /\.(ts|tsx)$/,
-      use: [{ loader: '@griffel/webpack-loader' }],
+      use: [
+        {
+          loader: '@griffel/webpack-loader',
+          options: {
+            babelOptions: {
+              presets: ['@babel/preset-typescript'],
+            },
```

Comment:
It was an implicit dependency before, https://github.com/microsoft/griffel/pull/55.